### DISCUSSION TITLE
Should (non-mineable) outfits only be saleable at planets with Outfitters?

### DISCUSSION THREAD

**Discussion by flowers9** [endless-sky/endless-sky#9059] [Balance, Mechanics, and Interactions]
Currently, if you have access to Trading, you can sell outfits from your cargo (as long as you're willing to sell them all), despite there being no Outfitter on the planet.  Having this ability causes some problems, for example it means parking ships on such planets can cause outfits to get orphaned (should there be insufficient space in the remaining fleet cargo capacity to hold them all).  It also doesn't match with how ships work, as those can only be sold at Shipyards.

If we removed the ability to sell non-mineable outfits on planets without Outfitters, we'd could leave them in fleet cargo rather than pooling them when you land, meaning they'd just stay on any ships you parked.  We could also do that for commodites and mineables on planets without Trading.  This would prevent some types of issues that show up when loading ships and not being able to load everything.

The main drawback (beyond ease of selling looted outfits) would be that you'd have to be careful about parking ships that carry mission cargo, rather than simply making sure your fleet has enough capacity for all your mission cargo.

**Comment by lumbar527** [comment]
What if every inhabited planet/planet with trading had an outfitter and shipyard, even if they were empty. If there are people on the planet, why can't you sell them stuff?

I think this would be another potential solution to some of the things you said.

**Comment by flowers9** [comment]
On a game level, it's nice to have some variety between worlds, giving the player reason to have to seek out particular places or facilities.  Having even empty shipyards and outfitters on every world might make things a bit too convenient.

Also, given the descriptions of a lot of planets, they simply don't have the facilities to handle a lot of stuff.  Not infrequently, spaceports are just marked off areas of dirt, and goods are transported by horse and wagon.  I mean, what would they even do with outfits you sell to them?

**Reply by lumbar527** [reply]
Maybe there's independent captains like you, happy to gather up some alien tech.

**Reply by lumbar527** [reply]
But then you would have to sell directly to the other captains, so never mind.

**Comment by Arachi-Lover** [comment]
I assume nearly all planets have at least some manner of scrapyard where you can sell whatever and they'll just dismantle it and use the materials in rudimentary ways.

**Comment by flowers9** [comment]
In a game balance sense, it would help make looting ships not quite as lucrative, as it'd be a bit harder to unload the non-commodity loot (which can be worth a lot more per ton than commodities).

Good point about the scrapyards, though.
